conda_smithy/templates/run_docker_build.sh.tmpl

```diff
@@ -53,15 +56,17 @@ DOCKER_RUN_ARGS=" -it "
 
 export UPLOAD_PACKAGES="${UPLOAD_PACKAGES:-True}"
 {{ docker.executable }} run ${DOCKER_RUN_ARGS} \
-           -v "${RECIPE_ROOT}":/home/conda/recipe_root:ro,z \
-           -v "${FEEDSTOCK_ROOT}":/home/conda/feedstock_root:rw,z \
-           -e CONFIG \
-           -e BINSTAR_TOKEN \
-           -e HOST_USER_ID \
-           -e UPLOAD_PACKAGES \
-           $DOCKER_IMAGE \
-           {{ docker.command }} \
-           /home/conda/feedstock_root/${PROVIDER_DIR}/build_steps.sh
+    -v "${RECIPE_ROOT}":/home/conda/recipe_root:ro,z \
+    -v "${FEEDSTOCK_ROOT}":/home/conda/feedstock_root:rw,z \
+    -e CONFIG \
+    -e BINSTAR_TOKEN \
+    -e HOST_USER_ID \
+    -e UPLOAD_PACKAGES \
+    $DOCKER_IMAGE \
+    {{ docker.command }} \
+    /home/conda/feedstock_root/${PROVIDER_DIR}/build_steps.sh
```

Comment:
Also are we able to revert the changes here too?

Reply:
we could, but this is just whitespace

Reply:
Exactly